src/scss/shared.scss

```diff
@@ -0,0 +1,6 @@
+// Background colors
+$primary-background-color: #67697C !important;
+$secondary-background-color: #C6878F !important;
+
+// Texts
+$primary-text: white !important;
```

Comment:
I think you accidentally took this change when you pulled. The `global.scss` file replaced this, so this `shared.scss` isn't used anymore. Let's delete it.